File: TimeFlies/vc/done/DoneTableViewCell.m
//
//  DoneTableViewCell.m
//  testTabbarController
//
//  Created by 林伟池 on 15/10/26.
//  Copyright © 2015年 林伟池. All rights reserved.
//

#import "DoneTableViewCell.h"

@implementation DoneTableViewCell

- (void)awakeFromNib {
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

#pragma mark - view init

- (void)viewInitWithTime:(NSDate *)endTime Text:(NSString *)text
{
    UILabel* timeLabel;
    UILabel* textLabel;

        timeLabel = self.myLeftText;
        textLabel = self.myRightText;
//    }
//    else{
//        timeLabel = self.myRightText;
//        textLabel = self.myLeftText;
//    }
    
    NSDateFormatter* dateFormatter = [[NSDateFormatter alloc] init];
    [dateFormatter setDateFormat:NSLocalizedString(@"yyyy年MM月dd日", nil)];

    timeLabel.text = [dateFormatter stringFromDate:endTime];
    textLabel.text = text;
}

- (void)clear
{
    self.myLeftText.text = @"";
    self.myRightText.text = @"";
}
#pragma mark - ui


#pragma mark - delegate

#pragma mark - notify


@end

File: TimeFlies/vc/done/DoneTableViewCell.swift
//
//  DoneTableViewCell.swift
//  TimeFlies
//
//  Created by 林伟池 on 16/4/8.
//  Copyright © 2016年 林伟池. All rights reserved.
//

import Foundation

class SwiftDoneTableViewCell: UITableViewCell {
    
    
    //MARK: property
    @IBOutlet var myLeftText:UILabel?
    @IBOutlet var myRightText:UILabel?
    
    
    //MARK: init
    func viewInitWithTime(_ endTime:Date, text:String) {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = NSLocalizedString("yyyy年M月dd日", comment: lyCommentDefault)
        myLeftText?.text = dateFormatter.string(from: endTime)
        myRightText?.text = text
    }
    
    
    //MARK: ui
    func clear() {
        myLeftText?.text = ""
        myRightText?.text = ""
    }
    
    //MARK: delegate
    
    
    //MARK: other
    
}
